pages/homepage.vue

```diff
@@ -0,0 +1,214 @@
+        </div>
+        <div class="hidden md:inline-block">
+          <img src="../assets/img/audiQ5.png" alt="audiQ5" class="audi w-4/5 h-auto">
+        </div>
+      </section>
+
+      <div class="px-0">
+        <div class="bg-white pt-8 px-4">
+
+            <div class="main flex flex-wrap md:ml-4 md:flex-no-wrap flex-col-reverse md:flex-row">
+              <div class="parent1 flex flex-col flex-no-wrap">
+                <div class="side-title flex mb-4">
+                  <div class="bg-accent-purple w-1 h-8"></div>
+                  <h1 class="text-accent-purple font-bold ml-4">NEW LISTINGS</h1>
+                </div>
+                <SideProductCard v-for="(sideItem, key) in sideItems" :key="key" :sideItem="sideItem"/>
+              </div>
+              <div class="parent2 flex flex-row flex-wrap md:mt-12">
+                <ProductCard v-for="(mainItem, key) in mainItems" :key="key" :mainItem="mainItem"/>
+              </div>
+            </div>
+
+        </div>
+      </div>
+
+      <div>
+        <Footer/>
+      </div>
+
+    </div>
+  </div>
+
+</template>
+
+<script>
+  import Nav from '~/layouts/nav.vue';
+  import Footer from '~/layouts/footer.vue';
+  import SearchBar from '~/components/SearchBar.vue';
+  import ProductCard from '~/components/GoodsCard.vue';
+  import SideProductCard from '~/components/SideGoodsCard.vue';
+  import AudiQ5 from '~/assets/img/audiQ5.png';
+
+    export default {
+      components: {Nav, Footer, SearchBar, AudiQ5, ProductCard, SideProductCard},
+      name: "homepage",
+      data() {
+        return {
+          sideItems:[
+            {
+              name: 'MASERATI',
+              model: 'Quattroprote',
+              transmission: 'auto',
+              price: '520.00',
+              src: require('~/assets/img/maserati.png')
+            },
+            {
+              name: 'ASTON MARTIN',
+              model: 'Vantage',
+              transmission: 'auto',
+              price: '720.00',
+              src: require('~/assets/img/aston.png')
+            },
+            {
+              name: 'FERRARI',
+              model: 'Spider 430',
+              transmission: 'auto',
+              price: '770.00',
+              src: require('~/assets/img/ferrari.png')
+            },
+            {
+              name: 'LAMBORGHINI',
+              model: 'Huracan',
+              transmission: 'auto',
+              price: '970.00',
+              src: require('~/assets/img/lambo.png')
+            },
+            {
+              name: 'MERCEDES-BENZ',
+              model: 'Slr',
+              transmission: 'auto',
+              price: '820.00',
+              src: require('~/assets/img/slr.png')
+            }
+          ],
+          mainItems:[
+            {
+              name: 'BMW',
+              model: '634Ci',
+              transmission: 'auto',
+              price: '480.00',
+              src: require('~/assets/img/635.png')
```

Comment:
Please, move images to /static folder and then you can use:
```src: '/img/635.png'``` without require.

Require not ok for images, because it increases size in the script.

Reply:
In some cases ```require``` acceptable for .svg but not for jpg/png.